javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/ApplicationLoggerBridge.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+public abstract class ApplicationLoggerBridge {
+
+  private static final AtomicReference<ApplicationLoggerBridge> applicationLoggerBridge =
+      new AtomicReference<>();
+
+  public static void set(ApplicationLoggerBridge bridge) {
+    ApplicationLoggerBridge.applicationLoggerBridge.compareAndSet(null, bridge);
+  }
+
+  public static void installApplicationLogger(InternalLogger.Factory applicationLoggerFactory) {
+    ApplicationLoggerBridge bridge = applicationLoggerBridge.get();
+    if (bridge != null) {
+      bridge.install(applicationLoggerFactory);
+    }
+  }
```

Comment:
The relationship between `set()` and `installApplicationLogger()` is a little confusing. I wish there was just one method...but I'm sure there are timing considerations that make this challenging.

Reply:
Yep, there need to be 2 steps to initialize this thing. The names could be better, but I didn't manage to think up anything less confusing. Suggestions welcome 😄